docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -1622,3 +1622,443 @@ heroku open
+
+``` yaml
+runtime: nodejs10
+
+instance_class: F2
+
+env_variables:
+  HOST: '<instance_id>.appspot.com'
+  NODE_ENV: 'production'
+  DATABASE_NAME: 'strapi'
+  DATABASE_USERNAME: 'postgres'
+  DATABASE_PASSWORD: '<password>'
+  INSTANCE_CONNECTION_NAME: '<instance_identifier>'
+
+beta_settings:
+  cloud_sql_instances: '<instance_identifier>'
+```
+
+:::
+
+::: tab Flexible Environment
+
+``` yaml
+runtime: nodejs10
+
+env: flex
+
+env_variables:
+  HOST: '<instance_id>.appspot.com'
+  NODE_ENV: 'production'
+  DATABASE_NAME: 'strapi'
+  DATABASE_USERNAME: 'postgres'
+  DATABASE_PASSWORD: '<password>'
+  INSTANCE_CONNECTION_NAME: '<instance_identifier>'
+
+beta_settings:
+  cloud_sql_instances: '<instance_identifier>'
+```
+
+:::
+
+::::
+
+Create `.gcloudignore` in the project root, include `app.yaml` here as well.
+
+```
+app.yaml
+.gcloudignore
+.git
+.gitignore
+node_modules/
+#!include:.gitignore
+```
+
+In the case of Strapi, the admin UI will have to be re-built after every deploy,
+and so we don't deploy local build artifacts, cache files and so on by including
+the `.gitignore` entries.
+
+### Configure the database
+
+The `PostgreSQL` database will need the `pg` package.
+
+``` bash
+yarn add pg
+```
+
+[Google App Engine requires](https://cloud.google.com/sql/docs/postgres/connect-app-engine) to connect to the database using the unix socket path, not an IP and port.
+
+Edit `database.json`, and use the socket path as `host`.
+
+```
+config/environments/production/database.json
+```
+
+``` json
+{
+  "defaultConnection": "default",
+  "connections": {
+    "default": {
+      "connector": "bookshelf",
+      "settings": {
+        "client": "postgres",
+        "host": "/cloudsql/${process.env.INSTANCE_CONNECTION_NAME}",
+        "database": "${process.env.DATABASE_NAME}",
+        "username": "${process.env.DATABASE_USERNAME}",
+        "password": "${process.env.DATABASE_PASSWORD}"
+      },
+      "options": {}
+    }
+  }
+}
+```
+
+Edit `server.json` to pick up the deployed hostname from the `HOST` variable in `app.yaml`.
+
+```
+config/environments/production/server.json
+```
+
+``` json
+{
+  "host": "${process.env.HOST}",
+  "port": "${process.env.PORT || 1337}",
+  "production": true,
+  "proxy": {
+    "enabled": false
+  },
+  "cron": {
+    "enabled": false
+  },
+  "admin": {
+    "autoOpen": false
+  }
+}
+```
+
+### Auto-build after deploy
+
+After deployment, the admin UI has to be re-built. This generates the contents of the `build` folder on the server.
+
+In `package.json`, add the `gcp-build` command to `scripts`:
+
+``` json
+{
+  "scripts": {
+    "gcp-build": "strapi build"
+  }
+}
+```
+
+### Deploy
+
+``` bash
+gcloud app deploy app.yaml --project myapi-123456
+```
+
+Watch the logs:
+
+``` bash
+gcloud app logs tail --project=myapi-123456 -s default
+```
+
+Open the admin page and register and admin user.
+
+```
+https://myapp-123456.appspot.com/admin/
+```
+
+### Creating new content types
```

Comment:
Hello! Can you please remove this section.
In other guides, we are not mentioning step to create content type because it's not the purpose of this guide.
Users already have their app ready to be deployed.